src/org-units-selector/OrgUnitsSelector.jsx

```diff
@@ -185,14 +187,18 @@ export default class OrgUnitsSelector extends React.Component {
         if (!this.state.levels) return null;
 
         const { levels, currentRoot, roots, groups } = this.state;
-        const { selected, controls } = this.props;
+        const { selected, controls, withElevation } = this.props;
         const { filterByLevel, filterByGroup, selectAll } = controls;
         const someControlsVisible = filterByLevel || filterByGroup || selectAll;
         const { renderOrgUnitSelectTitle: OrgUnitSelectTitle } = this;
         const initiallyExpanded = roots.length > 1 ? [] : roots.map(ou => ou.path);
         const getClass = root => `ou-root-${root.path.split("/").length - 1}`;
         const leftStyles = someControlsVisible ? styles.left : styles.leftFullWidth;
 
+        if (!withElevation) {
+            styles.cardWide.boxShadow = "none";
```

Comment:
As a rule of thumb, we tend to write immutable code. Here, there is also a practical reason, as `styles.cardWide` is used later as a prop. Proposal:

const cardWideStyle = {...styles.cardWide, boxShadow: "none"}
...
 <Card style={cardWideStyle}>
```

Reply:
ok, fixed!